src/sc/sidechain.h

```diff
@@ -198,7 +200,8 @@ class CSidechain {
                (this->balance                                    == rhs.balance)                           &&
                (this->fixedParams                                == rhs.fixedParams)                       &&
                (this->mImmatureAmounts                           == rhs.mImmatureAmounts)                  &&
-               (this->scFees                                     == rhs.scFees);
+               (this->scFees                                     == rhs.scFees)                            &&
```

Comment:
https://github.com/HorizenOfficial/zen/pull/478/files#diff-de2156b8f957c51e903f07895ddd1310f6925d1bf0646a66a5b16a8c3f2654feR192-R194:~:text=return%20(this%2D%3EcreationBlockHeight,(this%2D%3EscFees_v2

Can all `this->` be removed (as well as for `CSidechainEvents` `operator==` inside same file)

Reply:
I am not sure why "this" has been used in a similar context, but I saw it elsewhere in the code. We might clean up this - HA! - in a future refactoring.